specs/ChainSpec/LightClient.md

```diff
@@ -156,7 +163,70 @@ The sum of all the stakes of `next_bps` in the previous epoch is `total_stake` r
 
 The signatures in the `LightClientBlockView::approvals_after_next` are signatures on `approval_message`. The  `i`-th signature in `approvals_after_next`, if present, must validate against the `i`-th public key in `next_bps` from the previous epoch. `approvals_after_next` can contain fewer elements than `next_bps` in the previous epoch.
 
-## RPC end-point
+## Proof Verification
+
+[transaction-outcome-proof](Transaction Outcome Proofs)
+### Transaction Outcome Proofs
+
+To verify that a transaction or receipt happens on chain, a light client can request a proof through rpc by providing `id`, which is of type
+```rust
```

Comment:
Does it actually render properly without an extra line break before "```"?

Reply:
I think so